src/group_theory/free_group.lean

```diff
@@ -501,7 +501,7 @@ variables (X : Type*) (G : Type*) [group G]
 
 /-- The bijection underlying the free-forgetful adjunction for groups. -/
 @[simps]
-def hom_equiv : (free_group X →* G) ≃ (X → G) :=
+def lift : (free_group X →* G) ≃ (X → G) :=
```

Comment:
Thanks for the rename - but the convention for lift has to_fun and inv_fun swapped

Reply:
Done.